celestia/peers.txt

```diff
@@ -1 +1,2 @@
+b1187a51400cef8b49a8482088eac57327989fe8@135.181.75.114:26656
```

Comment:
please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`